.github/workflows/apidiff.yml

```diff
@@ -6,11 +6,29 @@ on:
   pull_request:
 
 jobs:
+  check_docs_only:
+    name: check_docs_only
+    runs-on: ubuntu-18.04
+    outputs:
+      skip: ${{ steps.check_docs_only.outputs.skip }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - id: check_docs_only
+        # Since PR's are squashed prior to merging to the branch checked out (default branch),
+        # HEAD^ will resolve to the previous point in history.
+        run: |
+          REF="HEAD^"
+          [[ -z "${{ github.base_ref }}" ]] || REF=$(git show-ref ${{ github.base_ref }} | head -1 | cut -d' ' -f2)
+          echo "::set-output name=skip::$(test/check-docs-only.sh $REF)"
```

Comment:
do we need to duplicate the code OR could it be centralized in one test only? Did we check that?